BUILDING.md

```diff
@@ -15,6 +15,15 @@ or MonoDevelop.
 
 The solutions all place their output in a common bin directory under the solution root.
 
+## Running Tests
+
+The tests that should be run in the solution are grouped by project name:
+ * `nunit.framework.tests-*`
+ * `nunitlite.tests-*`
+ * `slow-nunit-tests-*`
```

Comment:
IMO, you should know as a dev but not have to know as a user. The script knows what to test!